mmseg/models/backbones/twins.py

```diff
@@ -0,0 +1,632 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from mmcv.cnn import build_conv_layer, build_norm_layer, trunc_normal_init
+from mmcv.cnn.bricks.drop import build_dropout
+from mmcv.cnn.bricks.transformer import FFN
+from mmcv.runner import BaseModule, ModuleList
+
+from mmseg.models.backbones.mit import EfficientMultiheadAttention
+from mmseg.models.backbones.swin import WindowMSA
+from mmseg.models.builder import BACKBONES
+from ..utils.embed import PatchEmbed
+
+
+class GlobalSubsampledAttention(EfficientMultiheadAttention):
+    """global sub-sampled attention (Spatial Reduction Attention)
+        This module is modified from EfficientMultiheadAttention
+        which is a module from mmseg.models.backbones.mit.py
+    Args:
```

Comment:
```suggestion

    Args:
```

Add one blank line between description and args.